### ISSUE TITLE
No skills despite the default skill tree being installed

### ISSUE THREAD

**Issue by GitUser14975** [pufmat/skillsmod#95]
It seems I can't see any skills despite the default skill tree datapack being enabled.
puffish_skills-0.12.2-1.20-forge
![obraz](https://github.com/pufmat/skillsmod/assets/100159503/23f0e0e8-c696-46b8-8225-9d9a748d588d)
![obraz](https://github.com/pufmat/skillsmod/assets/100159503/29957b57-e3cb-4123-9a8a-a5286a8bc1c8)

**Comment by pufmat** [comment]
Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?

**Comment by makumaku1974** [comment]
> Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?

wait this is required now?

**Comment by makumaku1974** [comment]
> Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?

I installed both and nothing appeared too. but for 1.21

**Comment by GitUser14975** [comment]
> Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?

I didn't have it installed, I thought all dependencies installed themselves alongside the mode since I'm using the CurseForge launcher. Anyway, it works fine for me now, but I won't close the issue as makumaku1974 seems to still be having some problems with it

**Comment by makumaku1974** [comment]
> > Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?
> 
> I didn't have it installed, I thought all dependencies installed themselves alongside the mode since I'm using the CurseForge launcher. Anyway, it works fine for me now, but I won't close the issue as makumaku1974 seems to still be having some problems with it

Is it working for you now?

**Comment by GitUser14975** [comment]
> > > Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?
> > 
> > 
> > I didn't have it installed, I thought all dependencies installed themselves alongside the mode since I'm using the CurseForge launcher. Anyway, it works fine for me now, but I won't close the issue as makumaku1974 seems to still be having some problems with it
> 
> Is it working for you now?

Yeah, after adding Pufferfish's Attributes it started working normally. However I am using 1.20.1 and not 1.21 so you might be experiencing a different issue

**Comment by makumaku1974** [comment]
> > > > Did you install [Pufferfish's Attributes](https://www.curseforge.com/minecraft/mc-mods/puffish-attributes)?
> > > 
> > > 
> > > I didn't have it installed, I thought all dependencies installed themselves alongside the mode since I'm using the CurseForge launcher. Anyway, it works fine for me now, but I won't close the issue as makumaku1974 seems to still be having some problems with it
> > 
> > 
> > Is it working for you now?
> 
> Yeah, after adding Pufferfish's Attributes it started working normally. However I am using 1.20.1 and not 1.21 so you might be experiencing a different issue

I'll be reinstalling this mod to my modpack yet again, due to the errors of the Phase 1 modpack. Which means I'll be redoing everything with the said modpack and also keeping other devs in check because some mods that I added bloated it apparently and I'm still having a problem with some sort of Keybindings because I think I've downloaded a mod that allows me to combine Ctrl + Shift + Any Keybinds that I would like to use. Now, im stuck and miserable because I really dont know where did I get that mod tbh and it is 1.21. Been working already with this modpack for 2 weeks already and wishing every mods work as intended. So for now, if this Pufferfish's Skills work then i'll be using it and credit you guys.

**Comment by makumaku1974** [comment]
uhhh still the same issue for 1.21 for me.
![image](https://github.com/user-attachments/assets/f41ce9e0-cfa2-4bde-b352-e3f37897636d)

**Comment by pufmat** [comment]
After reloading or entering the world you should get a toast message:
- If it says "Invalid configuration, check game logs.", then check the logs and search for `puffish_skills`. 
  -  If in these logs there is something about attributes, then Pufferfish's Attributes is probably is not installed.
-  If it says "No skill trees provided, make sure to add one.", then the data pack is not installed or not enabled.

**Comment by makumaku1974** [comment]
I'll test this out now.

**Comment by makumaku1974** [comment]
> After reloading or entering the world you should get a toast message:
> 
> * If it says "Invalid configuration, check game logs.", then check the logs and search for `puffish_skills`.
>   
>   * If in these logs there is something about attributes, then Pufferfish's Attributes is probably is not installed.
> * If it says "No skill trees provided, make sure to add one.", then the data pack is not installed or not enabled.

Datapack is not installed.

**Comment by pufmat** [comment]
So you need to download the [Default Skill Trees](https://www.curseforge.com/minecraft/data-packs/default-skill-trees) or any other skill trees data pack. For more information about installing it check the [data pack installation tutorial on Minecraft Wiki](https://minecraft.wiki/w/Tutorials/Installing_a_data_pack).

**Comment by makumaku1974** [comment]
This works already thanks! but I think jump height should be removed.

**Comment by Caleb5904** [comment]
I have the same problem but it only happens on the server. in single player it works fine. I have all three mods installed and they are the same versions on forge and on the server

**Comment by makumaku1974** [comment]
> I have the same problem but it only happens on the server. in single player it works fine. I have all three mods installed and they are the same versions on forge and on the server

maybe client only mod? You could have checked the modrinth details i assume.
